python/lsst/drp/tasks/assemble_cell_coadd.py

```diff
@@ -323,6 +323,20 @@ def __init__(self, *args, **kwargs):
             self.makeSubtask("scale_zero_point")
 
         self.psf_warper = afwMath.Warper.fromConfig(self.config.psf_warper)
+        if "lanczos" in (warping_kernel_name := self.config.psf_warper.warpingKernelName.lower()):
+            psf_padding = 2 * int(warping_kernel_name.lstrip("lanczos")) - 1
+            self.log.debug(
+                "Padding PSF image by %d pixels since the warping kernel is %s.",
+                psf_padding,
+                warping_kernel_name,
+            )
+        else:
+            psf_padding = 10
```

Comment:
The only other non-lanczos option is bilinear; why 10 for that and only 5 for lanczos3?